### PR TITLE
Max election age precision

### PR DESCRIPTION
**Pull request by RickiNano**
`max_election_age` is currently being casted as `std::chrono::seconds` which causes the value to be rounded to nearest second. This pr changes it to use `double` and returns the value with 2 decimals.
It's unfortunate that I did not notice this issue in the original PR.
The unit test is also updated to test that both election age values are greater than zero. On my machine they are 0.09 seconds during unit test.
I also implemented the syntax Colin suggested for `oldest_election_start` to avoid the if statement

### CODE REVIEW

File: nano/rpc_test/rpc.cpp [hunk 1]
```diff
@@ -6901,7 +6901,7 @@ TEST (rpc, election_statistics)
 		ASSERT_EQ ("0", response.get<std::string> ("optimistic"));
 		ASSERT_EQ ("1", response.get<std::string> ("total"));
 		ASSERT_NE ("0.00", response.get<std::string> ("aec_utilization_percentage"));
-		ASSERT_NO_THROW (response.get<std::string> ("max_election_age"));
```
Comment (dsiganos) [inline nano/rpc_test/rpc.cpp:6904]: You could also check that the value is less than 5 seconds.

**Comment by dsiganos** [comment]
The unit test is a little messy and more complicated than it needs to be.
I will tidy it up so that you can see what are the unnecessary elements.

**Comment by dsiganos** [comment]
There is no need to disable_request_loop because there are no voters on the network and the election will not be voted on and it will sit and wait.

**Review by dsiganos — APPROVED** [review]

**Review by dsiganos — DISMISSED** [review]

**Review by clemahieu — DISMISSED** [review]

**Comment by simpago** [comment]
Wouldn't it be simpler to return integer milliseconds instead of floating point seconds?

**Comment by RickiNano** [comment]
Good point @simpago 
What do you think @dsiganos and @clemahieu ?

**Comment by dsiganos** [comment]
In general, I always favour integers and fixed point to floating point. Floating point comes with too many traps. I agree that returning milliseconds is preferable.

**Comment by RickiNano** [comment]
I have updated it to use milliseconds

**Comment by dsiganos** [comment]
But now the other floating point sticks out like a sore thumb...
We can avoid the other floating point by diving it into smaller units too.

**Comment by RickiNano** [comment]
I don't see anything wrong with having the percentage value as a floating point value. Without any decimals it lacks the required precision.

**Review by dsiganos — APPROVED** [review]

**Comment by dsiganos** [comment]
Floats are inaccurate and cause problems and it is good to avoid them. But  in RPC code, I don't mind having floats, I would be worried if the floats were in the protocol code.
